Tabloid/Controllers/PostController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.VisualBasic;
+using System.Collections.Generic;
+using System.Security.Claims;
+using Tabloid.Repositories;
+using System;
+
```

Comment:
Double check your 'using' statements, you for sure don't need Microsoft.VisualBasic